openshift_cli_installer/libs/clusters/ocp_cluster.py

```diff
@@ -63,7 +70,7 @@ def __init__(self, ocp_cluster, **kwargs):
                 "acm": self.cluster.get("acm") is True,
                 "acm-observability": self.cluster.get("acm-observability") is True,
                 "acm-observability-s3-region": self.cluster.get(
-                    "acm-observability-s3-region", self.cluster_info["region"]
+                    "acm-observability-s3-region", self.cluster_info.get("region")
```

Comment:
I would expect to have here region, either the user passed it or we auto set it.

Reply:
In this case, all ACM related info should be set only if `self.create`